scripts/rds-stage-refresh.sh

```
@@ -0,0 +1,533 @@
+    CLEANUP_NEEDED=false
+
+    log_success "terraform.tfvars 복원 완료"
+fi
+
+echo ""
+
+# Step 6: Automated Data Masking (GDPR/CCPA 컴플라이언스)
+log_info "Step 6: 데이터 마스킹 처리 중..."
+
+# 마스킹 대상 테이블/컬럼 설정 파일 경로
+MASKING_CONFIG="${TERRAFORM_DIR}/../../../configs/data-masking-config.json"
```

Shellcheck (0.11.0)

[warning] MASKING_CONFIG appears unused. Verify use (or export if used externally).

(SC2034)